src/components/standalone/routes/RoutesContent.vue

```diff
@@ -0,0 +1,418 @@
+      .finally(() => {
+        deleting.value = false
+        deleteRouteId.value = undefined
+        deleteRouteName.value = undefined
+      })
+  }
+}
+
+function scrollToMainTable() {
+  let element = document.getElementById('divMainTable')
+  if (element) {
+    element.scrollIntoView({
+      behavior: 'smooth',
+      block: 'start'
+    })
+  }
+}
+</script>
+
+<template>
+  <NeSkeleton v-if="loading" :lines="15" />
+  <HorizontalCard v-if="!loading && !routes.length" class="space-y-4 text-center">
+    <p>{{ t('standalone.routes.no_route_found') }}</p>
+    <NeButton :kind="'primary'" @click="openCreateRoute()">
+      <template #prefix>
+        <FontAwesomeIcon :icon="faCirclePlus" />
+      </template>
+      {{ t('standalone.routes.create_route') }}
+    </NeButton>
+  </HorizontalCard>
+  <div v-if="!loading && routes.length">
+    <div class="space-y-8">
+      <div class="flex">
+        <div>
+          <p class="text-sm text-gray-500 dark:text-gray-400">
+            {{ t('standalone.routes.route_description') }}
+          </p>
+        </div>
+        <NeButton
+          v-if="routes.length >= 1"
+          kind="primary"
+          class="ml-auto self-start"
+          @click="openCreateRoute()"
+        >
+          <template #prefix>
+            <FontAwesomeIcon :icon="faCirclePlus" />
+          </template>
+          {{ t('standalone.routes.create_route') }}
+        </NeButton>
+      </div>
+    </div>
+  </div>
+  <NeInlineNotification
+    v-if="error.notificationTitle"
+    class="my-4"
+    kind="error"
+    :title="error.notificationTitle"
+    :description="error.notificationDescription"
+  />
+  <div v-if="!loading && routes.length">
+    <div class="my-4">
+      <NeButton kind="tertiary" size="sm" class="-ml-2" @click="scrollToMainTable()">
+        {{ t('standalone.routes.route_toggle_table') }}
+      </NeButton>
+    </div>
+    <div class="space-y-6">
+      <NeSkeleton v-if="loading" :lines="3" :size="'sm'" />
+      <template v-if="!loading && routes.length > 0">
+        <NeTable
+          :data="routes"
+          :headers="[
+            {
+              key: 'ns_description',
+              label: t('standalone.routes.route_name')
+            },
+            {
+              key: 'interface',
+              label: t('standalone.routes.route_interface')
+            },
+            {
+              key: 'target',
+              label: t('standalone.routes.route_network_address')
+            },
+            {
+              key: 'gateway',
+              label: t('standalone.routes.route_gateway')
+            },
+            {
+              key: 'metric',
+              label: t('standalone.routes.route_metric')
+            },
+            {
+              key: 'status',
+              label: t('standalone.routes.route_status_table')
+            },
+            {
+              key: 'actions'
+            }
+          ]"
+          :loading="loading"
+        >
+          <template #tbody>
```

Comment:
This is fine, but that's the hard way: with NeTable you can specify a `template` for every table cell (like you did on the table below)